Remakes the Deathclaw Cave. (controversial) (sexy

magicianofwands opened the pull request:
THIS WILL CONFLICT WITH YOBROCHARLIES CURRENT CHANGES TO THE DEATHCLAW CAVE. HE NEEDS TO REVERT ALL CHANGES HE MADE TO THE DEATHCLAW CAVE.

The deathclaw cave is currently an extremely easy dungeon, with you needing to kill merely four deathclaws to get access to actual power armor. Insane, right? The dungeon is also very poorly designed, it just doesn't make any logical sense. Why the hell are deathclaws in a completely sealed cave with a bunch of cazadors inside with them? Cazadors and deathclaws arent' friends. Add that onto the current testmerged PR from charlie that stealthbuffed the cave, adding multiple health NVGs and two spawns for power armor? It's insane. 


This PR makes the deathclaw cave MUCH harder, Much, much, much harder. Multiple deathclaws in closed spaces, no barricades between the rooms. In addition, the cave is now logically consistent. (no random nightstalkers or cazadors here!). There is also a "story" with a brotherhood strike team being sent to clear out the cave. Judging by the power armor you can now find, I don't think that went too well, huh? 

I'm open to feedback about balance, but I'm really only going to take balance seriously from Tadano or w/e. Also if this merge conflicts I'm killing myself.

magicianofwands committed:
Remakes the Deathclaw Cave

:D

magicianofwands committed:
Adds a lore note or something idfk

a

magicianofwands committed:
last minute tweaks

a

kacperpl1234pl commented:
maybe add pa book for this many claws whoever doing cave deserve it

magicianofwands committed:
Nests this shit up yo

Places nest to get rid of some of the mobs so we have less lag idk

magicianofwands committed:
Cobwebs that shit up yo

Adds cobwebs in some of the walkways to make it harder to cheese, in accordance with the nesting commit

magicianofwands committed:
Replaces the "knight corpse spawners" with human corpse spawners.

This stops the enclave from jacking knight combat armor or some shit yo

CVSthePharmacy commented:
They don't deserve the PA book, they NEED the PA book for this to be SLIGHTLY worth. Cause rn this is looking like jorje levels of work to do.

magicianofwands committed:
last minute commit

a

Aurrain committed:
Merge branch 'master' into mapping5

Yobrocharlie commented:
> THIS WILL CONFLICT WITH YOBROCHARLIES CURRENT CHANGES TO THE DEATHCLAW CAVE. HE NEEDS TO REVERT ALL CHANGES HE MADE TO THE DEATHCLAW CAVE.
> 
> The deathclaw cave is currently an extremely easy dungeon, with you needing to kill merely four deathclaws to get access to actual power armor. Insane, right? The dungeon is also very poorly designed, it just doesn't make any logical sense. Why the hell are deathclaws in a completely sealed cave with a bunch of cazadors inside with them? Cazadors and deathclaws arent' friends. Add that onto the current testmerged PR from charlie that stealthbuffed the cave, adding multiple health NVGs and two spawns for power armor? It's insane.
> 
> This PR makes the deathclaw cave MUCH harder, Much, much, much harder. Multiple deathclaws in closed spaces, no barricades between the rooms. In addition, the cave is now logically consistent. (no random nightstalkers or cazadors here!). There is also a "story" with a brotherhood strike team being sent to clear out the cave. Judging by the power armor you can now find, I don't think that went too well, huh?
> 
> I'm open to feedback about balance, but I'm really only going to take balance seriously from Tadano or w/e. Also if this merge conflicts I'm killing myself.

I didn’t stealth buff it I directly said in my PR I bugged it

Yobrocharlie commented:
_However_
I agree with the changes done in this PR

Yobrocharlie commented:
> > > THIS WILL CONFLICT WITH YOBROCHARLIES CURRENT CHANGES TO THE DEATHCLAW CAVE. HE NEEDS TO REVERT ALL CHANGES HE MADE TO THE DEATHCLAW CAVE.
> > > The deathclaw cave is currently an extremely easy dungeon, with you needing to kill merely four deathclaws to get access to actual power armor. Insane, right? The dungeon is also very poorly designed, it just doesn't make any logical sense. Why the hell are deathclaws in a completely sealed cave with a bunch of cazadors inside with them? Cazadors and deathclaws arent' friends. Add that onto the current testmerged PR from charlie that stealthbuffed the cave, adding multiple health NVGs and two spawns for power armor? It's insane.
> > > This PR makes the deathclaw cave MUCH harder, Much, much, much harder. Multiple deathclaws in closed spaces, no barricades between the rooms. In addition, the cave is now logically consistent. (no random nightstalkers or cazadors here!). There is also a "story" with a brotherhood strike team being sent to clear out the cave. Judging by the power armor you can now find, I don't think that went too well, huh?
> > > I'm open to feedback about balance, but I'm really only going to take balance seriously from Tadano or w/e. Also if this merge conflicts I'm killing myself.
> > 
> > 
> > I didn’t stealth buff it I directly said in my PR I bugged it
> 
> Wrong terminology, but it was still out of the scope of your original PR imo

Yea

Yobrocharlie commented:
Okay my changes are undone

magicianofwands committed:
Merge branch 'master' into mapping5

Loldaddy5676 commented:
just a heads up, i just took all the loot without killing a single claw as a tribal since you can easily lure all the claws out into the nearby sewers and caves. in order to make this dungeon actully good you gotta start locking away the loot behind locked doors that needs a key from maybe the mother AND alpha claw so they don't cheese it.

kacperpl1234pl commented:
hit new claw cave today hard as hell at the end theres 2 PA armors no pa helmets for them they simply did not spawn i cheaked under bodys too

kacperpl1234pl commented:
also while at it this place really needs a pa book theres only 1 pa book spawn on the map except for alrem rigth now it be balanced since all the god damn claws that charge at you at once

magicianofwands committed:
This Should Be The Final Icing On The Cake

Makes the dungeon actually harder, makes the juvenile deathclaws actually smol, puts clothes on the dead bodies, adds a very smol baby deathclaw, and uhhh yea .

magicianofwands committed:
Fixes The Variable Issue

a

magicianofwands committed:
The Final Tweaks

i think its good now chief

magicianofwands committed:
I FORGOT TO REMOVE THE OIL

I FORGOT TO REMOVE THE OIL

magicianofwands committed:
tweaks the loot cause i placed the wrong spawners

aaa

kacperpl1234pl commented:
did claw cave now found 2 apa sets only 1 helmet yes 5 people cheaked whole cave for 2 pa helmet dont know whats up with that also maybe make sure theres at least one guarantted max level gun spawn we got sniper rifle and 2 watts out of cave today and we killed like 20 claws for that meanwhile 1 legendary claw in other dungeon hold like a plass pistol level gear

xTrainx committed:
Merge branch 'master' into mapping5

xTrainx committed:
Merge branch 'master' into mapping5

CVSthePharmacy commented:
> Stop commenting on github PRs when you have never coded, mapped, or sprited in your life :D
> 
> Also there are literally two power armor helmets. You're blind. Other people have found them.
> 
> No, it isn't APA, that's the wrong terminology, retard.
> 
> I have already pushed commits to slightly buff the loot drops but those changes aren't present in the testmerged version. I would need to yell at Tadano about that.
> 
> Leave me alone, I want an actual maintainer to give feedback and not you :D

don't code if you can't take feedback

xTrainx commented:
merge cenflecs
